bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Verify/VerifyElection.cs

```diff
@@ -0,0 +1,1087 @@
+using ElectionGuard.Decryption.ElectionRecord;
+using ElectionGuard.Decryption.Tally;
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.Verify;
+
+public static class VerifyElection
```

Comment:
this file is where all of the verification steps are run. This is the most interesting part of the pull request as it is what is responsible for checking everything.